libraries/anaconda3-solaris-0.1.2/context/solaris-build

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eu
```

Comment:
I don't mind doing it in this PR or some other but it would be nice to make the same change in `libraries/anaconda3-buildtime/context/build` and `libraries/anaconda3-runtime/context/pipe` so they also barf when we have an unbound ANACONDA_ENV variable